.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,4 @@
-blank_issues_enabled: false
 contact_links:
```

Comment:
should change info in this file to our link

Reply:
ok sweet, done